idaes/models_extra/power_generation/costing/tests/test_NGCC_costing.py

```diff
@@ -56,6 +49,15 @@ def test_get_costing(build_costing):
     m.fs.b1.feedwater_flowrate = pyo.Var(initialize=1085751)  # lb/hr
     m.fs.b1.feedwater_flowrate.fix()
     get_PP_costing(m.fs.b1, FW_accounts, m.fs.b1.feedwater_flowrate, "lb/hr", 6)
+
```

Comment:
Same question here on why this was moved - if `get_PP_costing` fails this will occur in the fixture now and not `test_get_costing`.